tests/mgr/test_restful_module_enable.py

```diff
@@ -0,0 +1,35 @@
+from cli.cephadm.cephadm import CephAdm
+from cli.exceptions import OperationFailedError
+from utility.log import Log
+
+LOG = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    ceph-mgr restful call for viewing/modifying cluster information
+    Args:
+        ceph_cluster (ceph.ceph.Ceph): ceph cluster object
+        kw: test config arguments
+    Returns:
+        int: non-zero on failure, zero on pass
+    """
+    config = kw.get("config")
+    node = ceph_cluster.get_nodes(role="mgr")[0]
+    username = config.get("username")
+
+    # Enable restful module
+    out = CephAdm(node).ceph.mgr.module.enable("restful")
+    if out:
```

Comment:
Yes, will correct it.

Reply:
Done

Reply:
Hey @vamahaja 
There is an issue, since the module is by default enabled, we see below output when I try to enable it again in https://github.com/red-hat-storage/cephci/pull/3990/files#diff-416a709e09db2a584198ae4058901a8d559bdebef4a87f721ec6a8d00ad692a5R22 ; 

`<class 'str'>
` 
which is empty string due to `out[0].strip()` which comes from -
https://github.com/sayaleeraut/cephci/blob/feb6c85ae8be1bba7c2501c50eb22be67e310f0a/cli/ceph/mgr/module.py#L12 

that returns something as below -

`<class 'tuple'>
('', "Inferring fsid 253172ee-6529-11ef-aa39-fa163e67161c\nInferring config /var/lib/ceph/253172ee-6529-11ef-aa39-fa163e67161c/mon.ceph-saraut-trw2qk-node
1-installer/config\nUsing ceph image with id 'f6647af01e82' and tag 'ceph-8.0-rhel-9-containers-candidate-71009-20240820205218' created on 2024-08-20 20:5
5:07 +0000 UTC\nregistry-proxy.engineering.redhat.com/rh-osbs/rhceph@sha256:eecaf15576d34d50c16aa5eeabecb4d8bf9068409ed7cedcd886a3dac1a914ae\nmodule 'rest
ful' is already enabled\n")`

Reply:
Will remove the validation, not required.